core/src/dbs/capabilities.rs

```diff
@@ -338,13 +338,6 @@ impl Capabilities {
 		self.live_query_notifications
 	}
 
-	// function is public API so we can't remove it, but you should prefer allows_function_name
-	pub fn allows_function(&self, target: &FuncTarget) -> bool {
-		self.allow_funcs.matches(target) && !self.deny_funcs.matches(target)
-	}
-
-	// doc hidden so we don't extend api in the library.
-	#[doc(hidden)]
 	pub fn allows_function_name(&self, target: &str) -> bool {
```

Comment:
I added the allows_function_name because when filtering out a function we are matching against the name of a function, a string. The previous function required you to parse a string first which was not required for the function to work. So I removed the parsing requirement. For net targets the target needs to be parsed for the function to work, so I thought it was better for that function to not take a string.